src/components/TodoComponent.tsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import classNames from 'classnames';
+import { Todo } from '../types/Todo';
+import { useEffect, useState } from 'react';
+import '../styles/todo.scss';
+type Props = {
+  todo: Todo;
+  onRemoveTodo: (id: number) => void;
+  isLoading?: boolean;
+};
+
+export const TodoComponent: React.FC<Props> = ({
+  todo,
+  isLoading,
+  onRemoveTodo,
+}) => {
+  const [todoClassName, setTodoClassName] = useState<string>(
+    'todo item-enter item-enter-active',
+  );
+  const [isToLoad, setIsToLoad] = useState(isLoading);
+
+  useEffect(() => {
+    //setTodoClassName(() => 'todo item-enter-done ');
+    // setTimeout(() => {
+
+    // }, 100);
+    return () => {
+      setTimeout(() => {
+        setTodoClassName(() => 'todo item-enter-done ');
+      }, 500);
+    };
+  }, []);
+
+  useEffect(() => {}, [isLoading]);
+
+  return (
+    <div
+      key={todo.id}
+      data-cy="Todo"
+      className={classNames(todoClassName, todo.completed && 'completed')}
```

Comment:
```suggestion
      className={classNames(todoClassName, { completed: todo.completed })}
```
Check how to use [classnames](https://www.npmjs.com/package/classnames) library, fix in all places